src/wallet/rpcwallet.cpp

```diff
@@ -4721,88 +4685,88 @@ extern UniValue importsaplingviewingkey(const JSONRPCRequest& request);
+        {"wallet", "sendtoaddress", &sendtoaddress, false, {"address", "amount", "comment", "comment-to", "subtract_fee"}},
+        {"wallet", "settxfee", &settxfee, true, {"amount"}},
+        {"wallet", "setstakesplitthreshold", &setstakesplitthreshold, false, {"value"}},
+        {"wallet", "signmessage", &signmessage, true, {"address", "message"}},
+        {"wallet", "walletlock", &walletlock, true, {}},
+        {"wallet", "walletpassphrasechange", &walletpassphrasechange, true, {"oldpassphrase", "newpassphrase"}},
+        {"wallet", "walletpassphrase", &walletpassphrase, true, {"passphrase", "timeout", "staking_only"}},
+        {"wallet", "rescanblockchain", &rescanblockchain, true, {"start_height", "stop_height"}},
+        {"wallet", "delegatoradd", &delegatoradd, true, {"address", "label"}},
+        {"wallet", "delegatorremove", &delegatorremove, true, {"address"}},
+        {"wallet", "bip38encrypt", &bip38encrypt, true, {"address", "passphrase"}},
+        {"wallet", "bip38decrypt", &bip38decrypt, true, {"encrypted_key", "passphrase"}},
+
+        /** Sapling functions */
+        {"wallet", "getnewshieldaddress", &getnewshieldaddress, true, {"label"}},
+        {"wallet", "listshieldaddresses", &listshieldaddresses, false, {"include_watchonly"}},
+        {"wallet", "exportsaplingkey", &exportsaplingkey, true, {"shield_addr"}},
+        {"wallet", "importsaplingkey", &importsaplingkey, true, {"key", "rescan", "height"}},
+        {"wallet", "importsaplingviewingkey", &importsaplingviewingkey, true, {"vkey", "rescan", "height"}},
+        {"wallet", "exportsaplingviewingkey", &exportsaplingviewingkey, true, {"shield_addr"}},
+        {"wallet", "getshieldbalance", &getshieldbalance, false, {"address", "minconf", "include_watchonly"}},
+        {"wallet", "listshieldunspent", &listshieldunspent, false, {"minconf", "maxconf", "include_watchonly", "addresses"}},
+        {"wallet", "rawshieldsendmany", &rawshieldsendmany, false, {"fromaddress", "amounts", "minconf", "fee"}},
+        {"wallet", "shieldsendmany", &shieldsendmany, false, {"fromaddress", "amounts", "minconf", "fee", "subtract_fee_from"}},
+        {"wallet", "listreceivedbyshieldaddress", &listreceivedbyshieldaddress, false, {"address", "minconf"}},
+        {"wallet", "viewshieldtransaction", &viewshieldtransaction, false, {"txid"}},
+        {"wallet", "getsaplingnotescount", &getsaplingnotescount, false, {"minconf"}},
+
+        /** Label functions (to replace non-balance account functions) */
+        {"wallet", "getaddressesbylabel", &getaddressesbylabel, true, {"label"}},
+        {"wallet", "getreceivedbylabel", &getreceivedbylabel, false, {"label", "minconf"}},
+        {"wallet", "listlabels", &listlabels, false, {"purpose"}},
+        {"wallet", "listreceivedbylabel", &listreceivedbylabel, false, {"minconf", "include_empty", "include_watchonly"}},
+        {"wallet", "setlabel", &setlabel, true, {"address", "label"}},
```

Comment:
unnecessary whitespace change. please preserve the table-style formatting for readability's sake.

Reply:
should be fixed now, clang-format-diff.py  automatically changed that